### ISSUE TITLE
Profile Image/myprofile  is not accessible by keyboard

### ISSUE THREAD

**Issue by ponnagouv** [ponnagouv/VidhyaTest#2]
Bug type: Accessibility Bug

**Steps to Reproduce:**
1.	Launch https://www.topcoder.com/ in Edge Browser
2.	Login with your id
3.	Navigate to the webpage using Tab key alone. 
4.	Press tab key in the Keyboard. Tab pass through topcoder, then Customer, then TAlent , then About us , then 9 dots image in Header section, then tab goes down to WHY TOPCODER button in the down. Tab did not reach profile image Next to 9dot image. 
**Actual Result**
1.	Launch https://www.topcoder.com/ in Edge Browser
2.	Login with your id
3.	Navigate to the webpage using Tab key alone. 
4.	Press tab key in the Keyboard. Tab pass through topcoder, Customer, TAlent , About us Titles and then 9 dots image in Header section, but next tab goes down to WHY TOPCODER button in the down. Tab did not reach profile image Next to 9dot image. 
***Expected Result:****
1.	Launch https://www.topcoder.com/ in Edge Browser
2.	Login with your id
3.	Navigate to the webpage using Tab key alone. 
4.	Press tab key in the Keyboard. Tab pass through topcoder, Customer, TAlent , About us Titles and then 9 dots image in Header section, but next tab should reach profile image Next to 9dot image, should be able to Access My Profile settings using keyaboard alone. 
As per WCAG Guidelines, Section 2.1 ( Operable-> Keyaboard Accessible - Make all functionality available from a keyboard) All the links/options should be available using keyboard alone. 
**Device/OS/Browser Information**
1.	OS: Windows 11 Enterprise 
2.	Browser : Edge

**Comment by ponnagouv** [comment]
Trying to attach the screenshot. Unable to upload file. Will try again
